DP3TApp/Screens/Meldungen/NSMeldungenDetailNoMeldungenTitleView.swift

```diff
@@ -15,7 +15,7 @@ class NSMeldungenDetailNoMeldungenTitleView: NSTitleView {
 
     private let stackView = UIStackView()
 
-    private let imageView = UIImageView(image: UIImage(named: "ic-check-circle"))
+    private let imageView = UIImageView(image: UIImage(named: "ic-check-circle")) // TODO: New icon with correct color
```

Comment:
This is a TODO that needs to be done? or did you just forget it?

Reply:
I removed the todo in the latest commit. Since we have some new colors, we need to redesign some of the icons that we cannot simply change the tint color of, and this was one of them.